testgrid/config/config.yaml

```diff
@@ -3540,9 +3542,15 @@ dashboards:
 
 - name: sig-big-data
   dashboard_tab:
-  - name: mlkube-gke
-    description: what does this tab do exactly
-    test_group_name: ci-kubernetes-e2e-mlkube-gke
+  - name: mlkube-build-periodic
+    description: Periodic builds and testing of TfJob CRD at head running on stable GKE.
+    test_group_name: mlkube-build-periodic
+  - name: mlkube-build-postsubmit
+    description: Postsubmit tests of TfJob CRD running on stable GKE.
+    test_group_name: mlkube-build-postsubmit
+  - name: mlkube-build-presubmit
+    description: Presubmit tests of TfJob CRD running on stable GKE.
+    test_group_name: mlkube-build-presubmit
   notifications:
   - summary: Please configure this skeleton dashboard and remove this notification
     context_link: https://github.com/kubernetes/test-infra/tree/master/testgrid/config
```

Comment:
(probably can delete this notification now)

Reply:
Done